stockflux-securities-master/src/components/SecuritiesTable.css

```diff
@@ -53,7 +59,43 @@
 .securities-exchange-data,
 .securities-symbol-data,
 .securities-name-data {
-  padding: var(--margin-medium) var(--margin-small);
+  padding: var(--margin-medium) var(--margin-large);
   border-bottom: 1px solid var(--clr-divider-light);
   border-left: 1px solid var(--clr-divider-light);
 }
+
+.no-securities-container {
+  border-bottom: 1px solid var(--clr-divider-light);
+  margin-top: calc(var(--table-body-height) * 0.3);
```

Comment:
Use flexbox instead of setting a margin-top

Reply:
Alright, done now